pkg/providers/tinkerbell/hardware/json.go

```diff
@@ -0,0 +1,156 @@
+package hardware
```

Comment:
i dont think we need to include tinkerbell changes?

Reply:
yes, I think this caused by merge vs rebase, this change is from main and both aws:cloudstack and wanyufe:controller has same change, somehow the PR still shows the diff.

Reply:
It just got pulled in because of the merge